File: java/src/warranty/TestContractAdmin.java
package warranty;

import org.junit.Test;

import warranty.Contract;

import junit.framework.Assert;

public class TestContractAdmin {
	
    @Test
    public void TestContractIsSetupCorrectly()
    {
        Contract contract = new Contract(0, 100.0);
        Assert.assertEquals(100.0, contract.purchasePrice);
        Assert.assertEquals(Contract.Status.PENDING, contract.status);
    }
    
}

File: php74/tests/Warranty/TestContractAdmin.php
<?php

declare(strict_types=1);

namespace DDDCodeSamples\Tests\Warranty;

use DDDCodeSamples\Warranty\Contract;
use PHPUnit\Framework\TestCase;

/**
 * Class TestContractAdmin
 *
 * @package DDDCodeSamples\Tests\Warranty
 */
class TestContractAdmin extends TestCase
{
    /**
     * @test
     */
    public function contractIsSetupCorrectly()
    {
        $contract = new Contract(0, 100.0);
        $this->assertEquals(100.0, $contract->purchasePrice);
        $this->assertEquals(Contract::PENDING, $contract->status);
    }
}
